src/neuroconv/nwbconverter.py

```diff
@@ -101,13 +101,20 @@ def get_metadata(self) -> DeepDict:
             metadata = dict_deep_update(metadata, interface_metadata)
         return metadata
 
-    def validate_metadata(self, metadata: Dict[str, dict]):
+    def validate_metadata(self, metadata: Dict[str, dict], append_mode: bool = False):
```

Comment:
Just something to keep in mind here

The argument on `run_conversion` is `overwrite` and it defaults to `False` (append being enabled by default if file exists)

I know you detect the file existence and whatnot below, but there's some slight API inconsistency here to think about

Reply:
Good point.